example/resource_virtual_environment_container.tf

```
@@ -85,7 +85,9 @@ resource "proxmox_virtual_environment_container" "example" {
 
   node_name = data.proxmox_virtual_environment_nodes.example.names[0]
   pool_id   = proxmox_virtual_environment_pool.example.id
-  vm_id     = 2043
+  # Set the protection flag to prevent the deletion/update operations for the container and its disks.
+  protection = true
```

```suggestion
  # protection = true 
```
well, this prevents container from removal, as expected, but also causes `make example` to fail :)